design/ceph/ceph-cluster-cleanup.md

```diff
@@ -0,0 +1,65 @@
+# Ceph cluster clean up policy
+
+## Use case
+
+As a rook user, I want to clean up the datadirhostpath after I intentionally uninstall ceph cluster , so that I can start a new cluster without having to do any manual clean up.
+
+## Background
+
+If the user deletes a cluster and starts a new cluster on the same hosts, the path used by dataDirHostPath must be deleted. Otherwise, stale keys and other config will remain from the previous cluster and the new mons will fail to start. As of now, the user has to manually delete the dataDirHostPath on each host. This implementation aims to automate this deletion.
```

Comment:
The issue of the re-install failing does not apply to clusters that are running on PVs. Let's call out that scenario as not needing this design since the PVs will be cleaned up already by K8s.